lib/components/primitive-components/Group/Group.ts

```diff
@@ -147,15 +149,32 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
       this.props.autorouter?.serverUrl ?? "https://registry-api.tscircuit.com"
     const serverMode = this.props.autorouter?.serverMode ?? "job"
 
-    const debug = Debug("tscircuit:core:autorouting")
+    // Find all child groups that are subcircuits
+    const subcircuits = this.children.filter(
+      (child) => child instanceof Group && child.isSubcircuit,
+    ) as Group[]
+
+    const subcircuitPromises = subcircuits.map((subcircuit) =>
+      subcircuit._queueAutorouting(serverUrl, serverMode, debug),
+    )
```

Comment:
this isn't quite right either, a subcircuit should only be responsible for itself